hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/HddsVersionInfo.java

```diff
@@ -52,6 +52,8 @@ public static void main(String[] args) {
             " and " + HDDS_VERSION_INFO.getHadoopProtoc3Version());
     System.out.println(
         "From source with checksum " + HDDS_VERSION_INFO.getSrcChecksum());
+    System.out.println(
+        "For platform " + HDDS_VERSION_INFO.getPlatform());
```

Comment:
Correct me if I'm wrong, I don't think the current Ozone binary package is platform specific.
If it's true, we should give information like "compiled on x86_64", but not "compiled for x86_64".

Reply:
Right, "compiled on" is an accurate wording.